### DISCUSSION TITLE
Trigger snapshot at the end of printing

### DISCUSSION THREAD

**Discussion by albatorsssx** [berrystephenw/OctoText#110] [Q&A]
Hello,

Is it possible to trigger the end of print snapshot before the G28 X0 Y0? Because the piece is out of frame. Otherwise what is the gcode that triggers the snapshot?

Thanks in advance.

Merry Christmas

**Comment by berrystephenw** [comment]
Sorry for the delay, I don't know exactly what triggers the end of print notification - it comes from Octoprint and is out of my control. This question has been asked before though. Try the solution given on this issue: [https://github.com/berrystephenw/OctoText/issues/84](url)
